e2e-test/src/test/java/de/sovity/edc/extension/e2e/connector/factory/TestConnectorFactory.java

```diff
@@ -0,0 +1,84 @@
+package de.sovity.edc.extension.e2e.connector.factory;
```

Comment:
license headers, and in many other files, too.

I'd search by regex it

Reply:
https://www.jetbrains.com/help/idea/copyright.html